R/endpoints.R

```diff
@@ -69,9 +68,7 @@ endpoint_model_result <- function(queue) {
       )
       response$value <- NULL
       res$status <- 400
-    } else if (response$success) {
-      response$value <- scalar(response$value)
-    } else {
+    } else if (!response$success) {
```

Comment:
I know it's not in this PR but I am surprised that the opening bit here is surprising that we're checking for an error class and not a success?